test/runner/tob.ts

```diff
@@ -0,0 +1,32 @@
+import { file2Tests } from './run';
+
+interface TestItem {
+  name: string;
+  input: any;
+  fn: Function;
+  options: any;
+  output: any;
+  $output: any;
+}
+// testing object
+export interface Tob {
```

Comment:
What's the plan with this `tob`?

Reply:
it stands a testing object parsed from a .md file. it could containing some testing items.

we just need 2 functions:
* parse md file to tob
* write tob to a file